daemon/src/routes_maker.rs

```diff
@@ -106,3 +110,32 @@ pub fn get_health_check() {}
 
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 struct RetrieveCurrentOrder;
+
+#[derive(RustEmbed)]
+#[folder = "../frontend/dist/maker"]
+struct Asset;
+
+#[rocket::get("/assets/<file..>")]
+pub fn dist<'r>(file: PathBuf) -> impl Responder<'r, 'static> {
+    let filename = format!("assets/{}", file.display().to_string());
+    Asset::get(&filename).map_or_else(
+        || Err(Status::NotFound),
+        |d| {
+            let ext = file
+                .as_path()
+                .extension()
+                .and_then(OsStr::to_str)
+                .ok_or_else(|| Status::new(400))?;
+            let content_type = ContentType::from_extension(ext).ok_or_else(|| Status::new(400))?;
+            Ok((content_type, d.data))
+        },
+    )
+}
+
+#[rocket::get("/<_paths..>")]
```

Comment:
Can we add `format = "text/html"` here?

That would make it more obvious that we only want to respond with `index.html` for all requests that ask for a HTML page.

I think  that should be possible: https://docs.rs/rocket/0.5.0-rc.1/rocket/attr.get.html#grammar